agriculture-nutrition/causes-pollution.html

```diff
@@ -0,0 +1,16 @@
+<div class="causes of soil pollution">
```

Comment:
Use shorter class name and remove spaces between words e.g. soil-pollution-causes